jmh/src/jmh/java/org/roaringbitmap/RadixSort.java

```diff
@@ -0,0 +1,48 @@
+package org.roaringbitmap;
+
+import org.openjdk.jmh.annotations.*;
+
+import java.util.SplittableRandom;
+
+@State(Scope.Benchmark)
+public class RadixSort {
+
+  @Param({"23", "25"})
+  int bits;
+
+  @Param("100000000")
+  int size;
+
+  @Param("0")
+  int seed;
+
+  private int[] data;
+  private int[] input;
+
+  @Setup(Level.Trial)
+  public void setup() {
+    SplittableRandom random = new SplittableRandom(seed);
+    data = new int[size];
+    input = new int[size];
+    for (int i = 0; i < size; ++i) {
+      int value = random.nextInt(1 << bits);
```

Comment:
To compare above and bellow 24bit numbers you may want to generate identical sequences and mask to the right number of bits.

Reply:
Could do, it would mean the low histograms are the same for the same seeds, but I don't expect there to be any bias in the range either way. The important thing, which is taken care of, is to control the seed.

Reply:
I will implement this suggestion later, it should allow for much more precise comparison, and should help confirm the data dependency theory.